src/test/java/org/broadinstitute/dsde/consent/ontology/service/MockTranslationService.java

```diff
@@ -15,6 +15,11 @@ public String translateDataset(String dataUse) {
         return "translated dataset";
     }
 
+    @Override
+    public String translateParagraph(String paragraph) {
+        return "{\"http://purl.obolibrary.org/obo/DUO_0000007\":{\"title\":\"Disease Specific Research\",\"category\":\"Data Use Permission\"}}";
```

Comment:
It would probably be better to make the tests independent of the actual `search-terms.json`. You'd have to make `Translate` injectable and make the json it reads in a configuration object.